generators/JSONtoTestsCoverage.jl

```diff
@@ -0,0 +1,33 @@
+# NOTE: This code is purely to test the coverage of the JSON to test converter.
```

Comment:
could you rename the file to `test_JSONtoTests.jl` to make it more clear? When I read coverage, I think of coverage reporting like on codecov and not the tests themselves.

Reply:
Sure, I'll do that in the next commit.